src/App.jsx

```diff
@@ -24,7 +24,7 @@ const App = () => (
           />
         </Routes>
       </main>
-      <Footer logo={process.env.LOGO_POWERED_BY_OPEN_EDX_URL_SVG} />
```

Comment:
Mind also removing the variable from https://github.com/openedx/frontend-app-gradebook/blob/master/.env.development#L10?